README.md

```diff
@@ -6,3 +6,17 @@ The relevant documentation can be found [here](https://docs.geteppo.com/feature-
 
 The current version supports Dot Net 7.0 Runtime.
 
+### Contributing (OSX)
+
+* Download dotnet 7.0 installer to have access to the cli and runtimes
+* Download dotnet binary and copy it to your $PATH
```

Comment:
took me almost 2 hours to get the environment right :-O

Reply:
😞 

Thanks for persevering and making it easier for the next developer. That sounds incredibly frustrating.